### ISSUE TITLE
No sentries but I have valid sentries inside the sentry folder

### ISSUE THREAD

**Issue by Floraly** [Marc3842h/Titan#63]
## Description

On Titan 1.5.0 Release its working! But when I compile it from source it says something like this!
```
Received on connected: ConnectedCallback {JobID=JobID {SequentialCount=...
No Sentry file found. Titan will ask for a confirmation code...
Logging in with Auth Code: null / 2FA Code: null / Hash: null
```

## Reproduction Steps

1. I copied the Accounts.json and Sentry folder to my new compiled Titan.
2. I tried to commend a friend with one of my Accounts which is SteamGuard protected and currently NOT used or logged in with real steam client!
3. I get this error on all my other Account which worked on Titan 1.5.0

## Errors

```
No Sentry file found. Titan will ask for a confirmation code...
There is no entry box popping up like in release 1.5.0 I can not enter any confirmation code here.
```

## System Information

**Operating System**: Problem exists on Windows, Linux and MacOS.
**Titan version**: Titan Version actual Titan source from github.

**Comment by Marc3842h** [comment]
Bug is known as has probably something to do with the disconnection method rework I introduced a few commits ago. Will fix hopefully soon.

**Comment by Marc3842h** [comment]
Fixed with 77fba185064ed16bbf845e170315724e5a83faf8.
